### ISSUE TITLE
Ability Window Resize on Move

### ISSUE THREAD

**Issue by DocileThief** [txj-xyz/rs3-ability-tracker#48]
Removing this line from line 24 of AbilityWindow.js on dev branch :
```js
windows.ability?.on('moved', this.updateSize);```

This fixed the issue. It no longer resizes when it is moved. 

Stemming from this issue > I think that it resizes the panels 4 pixels too much when resizing the window. (line 25 of the same code.) 
Would this need to be fixed/attended to?

**Comment by DPulavarthy** [comment]
Currently the `windows.ability?.on('moved', this.updateSize);` calls a function that updates the location of the ability tracker and stores it in the config. 
```js
    updateSize = _ => {
        // get position
        const [x, y] = windows.ability.getPosition();
        config.abilityWindow.x = x;
        config.abilityWindow.y = y;
        this.fixBounds();
```

If this code were to be removed then the tracker would no longer "save its position" when the tracker window or the app is restarted.

We have tested many different scenarios. We have tried splitting the events so that moving would not resize and resizing would not move it, this works fine most of the time but gets buggy when the `numberOfIcons` slider is used. The `dev-rework` branch does have some commits of various attempts to fix this issue (28c61266a771395269edab603cdebc26be04cb51, a4fbad2c56f54b7c6e887526224d40f9fd590ee5, 3e1322351680de5e416b22259bef8f4deb09bda3, dbff674ab017c48d5169ac1abd3bfd9ce639a43b), but minimizing the chances of this happening is the best we can do right now.

The best course of action would be to add a slider like the `numberOfIcons` slider on the main window for resizing to minimize the chances of the tracker bugging out when moving.

TLDR: It fixes the issue (sort of), but causes other ones.

**Comment by txj-xyz** [comment]
It appears that the config doesn't want to update due to me not checking before the value is updated. I fixed it in my local branch. Will push when i have time.
